genericapi/handler.go

```diff
@@ -84,15 +92,28 @@ func (h *Handler) ServeCreateAlert(w http.ResponseWriter, r *http.Request) {
 
 	summary := r.FormValue("summary")
 	details := r.FormValue("details")
+	action := r.FormValue("action")
 
-	summary = validate.SanitizeText(summary, alert.MaxSummaryLength)
-	details = validate.SanitizeText(details, alert.MaxDetailsLength)
+	contentType := r.Header.Get("Content-type")
+	if contentType == "application/json" {
```

Comment:
Let's use the ParseMediaType helper to handle any variations/encoding info:
```suggestion
	ct, _, err = mime.ParseMediaType(r.Header.Get("Content-Type"))
	if err != nil {
		http.Error(w, err.Error(), http.StatusUnsupportedMediaType)
		return
	}
	
	if ct == "application/json" {
```